ledger-core/v2/virtual/integration/object_test.go

```diff
@@ -57,7 +57,13 @@ func TestInitViaCTMethod(t *testing.T) {
 
 	select {
 	case <-requestIsDone:
-	case <-time.After(10 * time.Second):
+		require.Failf(t, "", "SM Object needs to wait until sm.waitGetStateUntil")
```

Comment:
Please, add unit test that will check for this.

Moreover, that test may be flaky. If system scheduler will pause thread executing goroutine - we'll accept result sooner and it'll fail.